### DISCUSSION TITLE
Could not resolve host Error

### DISCUSSION THREAD

**Discussion by blackface13** [supabase/supabase#28924] [Questions]
Hello everyone

Please help me, I make app by Unity for android, when app start and connect to supabase server, very many my users error with 
1. Fatal Exception: java.lang.Exception
SocketException : Could not resolve host '***********.supabase.co'

or 
 
2. Fatal Exception: java.lang.Exception
SocketException : mono-io-layer-error (111)

**Comment by silentworks** [comment]
There isn't much context provided here. You need to share the code where this issue is happening or provide more than these error messages.

**Comment by blackface13** [comment]
I'm use C# client libraries to Initialize supabase, but receive error

this my code, same docs: https://supabase.com/docs/reference/csharp/installing

> var url = Environment.GetEnvironmentVariable("SUPABASE_URL");
var key = Environment.GetEnvironmentVariable("SUPABASE_KEY");
var options = new Supabase.SupabaseOptions
{
    AutoConnectRealtime = true
};
var supabase = new Supabase.Client(url, key, options);
await supabase.InitializeAsync();

Not all of my users are having this error, but a lot of them are.
